qml/FileListPage.qml

```diff
@@ -157,7 +157,7 @@ Page {
 
             Image {
                 id: icon
-                property string fileMimeType: window.mimeToIcon(model.fileMimeType)
+                property string fileMimeType: MimeFormatter.mimeTypeToIcon(model.fileMimeType)
```

Comment:
Thanks for dividing the commit into two. Just here, this MimeFormatter should go to the other commit I guess (or I missed something).